src/main/java/org/openjfx/API2Semestre/database/Data.java

```diff
@@ -0,0 +1,59 @@
+package org.openjfx.api2semestre.database;
+
+import java.sql.Date;
```

Comment:
```suggestion
import java.util.Date;
```  

Utilizando java.util.Date em vez de java.sql.Date resolve o problema de conversão banco->objeto.  

**Fonte:**
`"java.lang.ClassCastException: class java.sql.Timestamp cannot be cast to class java.sql.Date (java.sql.Timestamp and java.sql.Date are in module java.sql of loader 'platform')"`
(https://stackoverflow.com/questions/25892417/caused-by-java-lang-classcastexception-java-sql-timestamp-cannot-be-cast-to-ja)  

O arquivo continua sendo salvo apenas com o cabeçalho tho

Reply:
interessante

Reply:
não consigo commitar